imports/api/rest/post-process-db-change-message.js

```diff
@@ -55,18 +55,18 @@ export default (req, res) => {
 
     case 'case_new_message':
       // https://github.com/unee-t/lambda2sns/issues/5
-      recipients = lookup([message.assignee_user_id, message.current_list_of_invitees].join(","))
+      recipients = lookup([message.new_case_assignee_user_id, message.current_list_of_invitees].join(','))
       recipients.forEach(to => {
-        sendEmail(to, 'caseNewMessage', caseNewMessageTemplate(to, caseTitle, caseId, messsage.message_truncated))
+        sendEmail(to, 'caseNewMessage', caseNewMessageTemplate(to, caseTitle, caseId, lookup(message.created_by_user_id)[0], message.message_truncated))
       })
       break
 
     case 'case_updated':
       // https://github.com/unee-t/lambda2sns/issues/4
       // More are notified: https://github.com/unee-t/lambda2sns/issues/4#issuecomment-399339075
-      recipients = lookup([message.assignee_user_id, message.case_reporter_user_id, message.current_list_of_invitees].join(","))
+      recipients = lookup([message.assignee_user_id, message.case_reporter_user_id, message.current_list_of_invitees].join(','))
```

Comment:
@kaihendry shouldn't this read 

`recipients = lookup([message.new_case_assignee_user_id, message.case_reporter_user_id, message.current_list_of_invitees].join(','))`

instead?

Reply:
oops: apparently it has been fixed in https://github.com/unee-t/frontend/pull/312/commits/ca57c718ebccb6b98d8351a9cf1c58c3e113a104